ROADMAP.md

```diff
@@ -0,0 +1,31 @@
+# Style Guides Roadmap
+
+> tl;dr The goal of this document is to document the priority items for style guide development over the next 3-6 months as well as key activities designed to grow awareness and adoption of the Style Guides at STScI.
+
+## Planned releases
+
+### v1.0 release (proposed release date: 3rd April 2020)
+
+Initial release of STScI Style Guides including:
+
+- Accessibility guide
+- GitHub repositories guide
+- Jupyter Notebooks guide
+- Python (naming, coding standards, testing etc.) guide
+- Python packaging guide
+- Security guide
+- Guidelines for how to advertise compliance with STScI Style Guides (https://github.com/spacetelescope/style-guides/issues/102)
+
+### v1.1 release (release date June 2020)
+
+- Merge/reconcile [INS standards for JWST community software](https://github.com/spacetelescope/ins-jwst-community-software) with STScI Style Guides.
+- Web pages guide
+- Extended security guide
+
+## Other key activities
```

Comment:
Some other things we might want to address:

- Figure out who the actual maintainers are of this.
- How we decide when to add a new guide.

Reply:
Getting that mission support (mentioned below) would be essential in recruiting and retaining maintainers, but that is a separate conversation.